nav2_bringup/launch/nav2_params.yaml

```diff
@@ -31,28 +82,81 @@ DwbController:
     PathDist.scale: 32.0
     GoalDist.scale: 24.0
     RotateToGoal.scale: 32.0
-local_costmap:
-  local_costmap:
-    ros__parameters:
-      robot_radius: 0.092
-      obstacle_layer:
-        enabled: False
-      always_send_full_costmap: True
-      observation_sources: scan
-      scan:
-        topic: /scan
-        max_obstacle_height: 2.0
-        clearing: True
-        marking: True
-global_costmap:
-  global_costmap:
-    ros__parameters:
-      obstacle_layer:
-        enabled: False
-      always_send_full_costmap: True
-      observation_sources: scan
-      scan:
-        topic: /scan
-        max_obstacle_height: 2.0
-        clearing: True
-        marking: True
+
+dwb_controller_local_costmap:
```

Comment:
Why the node name change? I also don't see anywhere in the launch file where you made an actual change to the node name.

We also want to keep the costmaps in separate namespaces otherwise their topics will interfere with each other.

Reply:
To sort out for my self the various node names and parameters for myself, I used a <node_name>_<composed_node_name> scheme. I can change the name back. 

Why would I have to change the node name in the launch file? As long as the name in the params yaml file matches the node name, you're all set. 

Good point about the namespaces. Let's talk so I can make sure to update this correctly.

Reply:
I'd like to fix the node names in a separate PR since I'll also have to update nav2_dwb_controller and nav2_navfn_planner in concert with this.